javascript/operations.js

```diff
@@ -18,7 +18,7 @@ export default {
       before(element, operation)
       operate(operation, () => {
         const { html, focusSelector } = operation
-        element.insertAdjacentHTML('beforeend', html || '')
+        element.insertAdjacentHTML('beforeend', html != null ? html : '')
```

Comment:
Curiosity: why did you not go for the `??`
https://caniuse.com/mdn-javascript_operators_nullish_coalescing

```js
        element.insertAdjacentHTML('beforeend', html && '')
```

Reply:
Ha! I wish I had a dollar....

TL;DR: esbuild seems to choke on it depending on vague things. Can't trust it. Same with safe navigation. Will revisit topic in ~2025.